### DISCUSSION TITLE
New FBA Inbound API v2024-03-20 - Impossible to ship pallets to more than one center when box information is known up front?

### DISCUSSION THREAD

**Discussion by MetaMan2023** [amzn/selling-partner-api-models#4398] [General]
It seems more and more clear there are serious technical aspects missing  from the API and that for the "left hand process" (where you know box content info up front) that it's impossible (or nearly impossible) to send to more than one center -- especially if you want to use LTL.

**Can someone please solve this for me:**

* Assume we are using "box information is known up front" (IE: Left hand path)
* We pack our boxes and create an inbound plan and provide Amazon our box weights, dimensions, sizes and packing info (SKU, QTY etc)  (STEP 3)
* Amazon returns a number of split options -- for example it lets us choose 1 center, 2 centers, 5 centers... (STEP 5.1 ListPlacementOptions)
* We choose an option to send to TWO centers

Now if we are shipping UPS ground -- no problem**.  Every box is unique and gets it own UPS labels. 

But what if we want to ship LTL?    In step 5.2 (Generate transportation options) we need to provide Pallet weights, dimensions and sizes.

Can someone explain to me how to know which box (that we provided in Step 3) is going to which center?

For example if I have boxes:

#1 #2 #3 #4 #5 #6

And I need to build two pallets, one to center A and one to center B ...

How do I know when 3 boxes go on pallet A and which 3 boxes go on pallet B?...  This seems impossible to do under the current system unless every box is identical.

There is a "listShipmentItems" function (** WHICH IS NOT TECHNICALLY LISTED IN THE LEFT HAND PROCESS BUT SEEMS TO FUNCTION AFTER "Generate placement options") -- but this call only returns the SKU + QTY going in that shipment, not the BOXES that are going in that shipment.

**Comment by puppsupr** [comment]
Hi,
You can call [listShipmentBoxes](https://developer-docs.amazon.com/sp-api/docs/fulfillment-inbound-api-v2024-03-20-reference#listshipmentboxes) before confirming the placement and transportation which gives you boxes in that shipment so u know the placement option where its going.

**Reply by MetaMan2023** [reply]
@puppsupr 

Thank you -- yes this function listShipmetnBoxes seems to be the solution.  It is providing the Box Ids which we can use to match boxes to build pallets.
